src/langchain_google_alloydb_pg/alloydb_vectorstore.py

```diff
@@ -731,9 +731,13 @@ def max_marginal_relevance_search_with_score_by_vector(
         return self.engine._run_as_sync(coro)
 
     async def set_maintenance_work_mem(self, num_leaves: int, vector_size: int) -> None:
-        index_memory_required = 50 * num_leaves * vector_size * 4
+        # Required index memory in MB
+        buffer = 1
+        index_memory_required = (
+            round(50 * num_leaves * vector_size * 4 / 1024 / 1024) + buffer
```

Comment:
Can you add a comment that this is converting bytes to MB?

Reply:
added